app/tests/mock/test_mock_sentence_transformer.py

```diff
@@ -24,3 +27,28 @@ def dot_product(v1, v2):
     assert dot_product(long_text, long_text) > dot_product(long_text, medium_text)
     assert dot_product(long_text, medium_text) > dot_product(long_text, short_text)
     assert dot_product(medium_text, medium_text) > dot_product(medium_text, short_text)
+
```

Comment:
Should we update the lines above this to use utils.dot_score instead of the custom dot product function?

Reply:
I kinda like that we've tested calculating the score in different ways:
- `dot_product` uses basic math
- `util.dot_score` uses tensor math (linear algebra)
- `.max_inner_product` uses pgvector's sqlalchemy implementation

These different methods could be useful to validate scores later.
With that said, I'm ambivalent since it's easy enough to replicate this work.